src/main/java/core/basesyntax/SquareSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class SquareSupplier {
+    private Random random = new Random();
+    private RandomSize size = new RandomSize();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomSquare() {

```

Comment:
Don't begin method implementation with an empty line. The empty line after the method signature should be removed.